code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -608,25 +597,13 @@
 /datum/reagent/medicine/morphine
 	name = "Morphine"
 	id = "morphine"
-	description = "A painkiller that allows the patient to move at full speed even in bulky objects. Causes drowsiness and eventually unconsciousness in high doses. Overdose will cause a variety of effects, ranging from minor to lethal."
+	description = "A sedative. Causes drowsiness and eventually unconsciousness in high doses. Overdose will cause a variety of effects, ranging from minor to lethal."
 	reagent_state = LIQUID
```

Comment:
It seems weird for morphine's effects to not match the real-world effects, given it's name.

Coffee countering it was an amusing joke with no real reason to take out, but now it's basically just another sleepy-time chem (we already have a number of those, I don't see having it at all post this time having any worth).

Reply:
Morphine serves as a superior med-X as it negates the slowdown from injuries but metabolizes far slower. Anyone could abuse it if they knew how to make it.

Reply:
That addresses none of my actual point.

Reply:
what's your point then-morphine as it stands is a disgustingly powerful, cheaper and easier to obtain alternative to med-x that has no downsides if used appropriately so it got something that negated its abilities removed


you did the same with turbo and meth, as meth no longer provides a speedboost to allow turbo to have a reason to exist

Reply:
#537 
>I am by no means advising adding more of different kinds of chemicals. Effects should be more or less unique, or have specific drawbacks or gains to emphasize the difference [between chemicals].

This nerf to morphine makes it so we don't really need to have morphine at all as it's yet another sleepy chemical. Secondly, morphine doesn't actually make you sleepy. It's a real world painkiller.

My point of contention wasn't any of your changes, it was that the scope wasn't sufficient, and could do with some name changes - entirely replacing the chemical with Med-X - etc, although that is more work.

Reply:
There I just changed the name to Ether. Everything else is exactly the same because now it's just a renamed sleep toxin, because I don't want to go through the effort of entirely removing morphine and changing multiple recipes but its current effects were too strong to let stand.

If you won't merge it regardless of that then i'll just undo the morphine changes and you can keep your morphine as is.

Reply:
Future PR to remove sleep toxin then. Or just merge the two.